### DISCUSSION TITLE
Auth user email rate limits

### DISCUSSION THREAD

**Discussion by hardenb** [supabase/supabase#16040] [Questions] [answered]
I have submitted this question to support@supabase.io but since this problem is stopping the launch of my app I will ask here as well in case I get a faster response.

I have a supabase project. I have the Pro version ($25) I recently added my custom SMTP server to my supabse project. I am using magicLink for authentication. I decided to use resend for my smtp server. I know the server does work because I have sent successful signUp Emails with it and I can see the emails come though both the supabase logs and now the resend logs. However after just 4 emails in an hour (The new supabase email rate limit) I get a 429 error. This error appears on the supabase platform and nothing makes it to the resend logs. I found documentation for resend API that says the their rate limit is 10 emails per second (https://resend.com/docs/api-reference/introduction#rate-limit) which would be 3600 per hour. This is the value I set as my rate limit for my smtp settings in supabase. However that is not reflecting in my app. I still can only login or signup about 4 users per hour.

Has anyone experienced this? I don't know if there is a setting in supabase somewhere that I need to change to remove the old rate limit. Anything is helpful. Thanks.

**Comment by GaryAustin1** [comment]
https://github.com/orgs/supabase/discussions/15896
Has more info.
But the new limit is 4 per hour when using their SMTP.
The solution is to do a custom SMTP then you have control of that limit and a better email delivery service.

**Reply by hardenb** [reply]
Thank you for the response. I am aware of the new rate limit. I have implemented the "solution" of a custom SMTP which I have control over the rate limit. I am still getting a 429 error from supabase. (I have check the logs and this is not the  third party SMTP server 'resend' who is hitting a rate limit. it is supabase)

**Reply by GaryAustin1** [reply]
You may want to generate an issue in supabase/supabase or contact support.  Not seeing this reported in issues that I've looked at around this.  It is odd it is 4ish as the claim is that is only for Supabase SMPT and your setting is higher.  If it was still stuck at 4 for all, the users with the rate limit issue would still be complaining after going to custom SMTP.

**Reply by hardenb** [reply, marked as answer]
I did create an issue. Which was helpful. Turns out there was a small bug in the frontend code that was making my SMTP settings appear to disappear even saving. Tech support helped get it fixed which helped me debug better. In the end restarting my DB and toggling "Allow users to signin" off and thenn back on seemed to have fixed the issue.
